cypress.config.js

```diff
@@ -0,0 +1,17 @@
+const { setupNodeEvents } = require('@tybalt/cypress');
```

Comment:
Can we move this into packages/cypress? https://docs.cypress.io/guides/guides/command-line#cypress-run-config-file-lt-configuration-file-gt